dist/rpm/openQA.spec

```diff
@@ -504,7 +506,12 @@ fi
 %service_del_preun openqa-continuous-update.timer
 
 %postun
-%service_del_postun %{openqa_services}
+# reload main service (but do not restart it via service_del_postun to minimize downtimes)
+if [ -x /usr/bin/systemctl ] && [ $1 -ge 1 ]; then
+    /usr/bin/systemctl reload %{openqa_main_service} || :
+fi
```

Comment:
should `openqa_main_service` be in `service_del_postrun` on systems where systemd might not be present? (e.g. containers)

Reply:
I don't think it is relevant at all without systemd. We also already use the same approach for worker services where it is good enough.